script.js

```diff
@@ -0,0 +1,36 @@
+name  = prompt("Hey, what's your name?")
+age = prompt("How old are you?")
+
+
+
+for(var i = age; i <= 17; i++) {
+name = "Young person (you have played the lottery illegally)";
+}
```

Comment:
I love that you used a loop, unfortunately this loop isn't really doing much. An example of something potentially more useful:
```js
while (age < 17) {
  alert("You are not old enough, come back when you are older")
  age = prompt("How old are you?")
}
```
This will loop until the user enters an age that is 18 or older